lib/src/main/java/uk/gov/di/ipv/cri/address/library/models/CanonicalAddress.java

```diff
@@ -2,42 +2,56 @@
 
 import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
 import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBDocument;
+import com.fasterxml.jackson.annotation.JsonAlias;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import uk.gov.di.ipv.cri.address.library.models.ordnancesurvey.Result;
 
+import java.util.Date;
 import java.util.Optional;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @DynamoDBDocument
 public class CanonicalAddress {
-    private String uprn;
+    private Optional<Long> uprn;
```

Comment:
Do you need this to be Optional, you could Optionalise the getter instead

Reply:
Done!